docs/content/development/extensions-core/avro.md

```diff
@@ -4,7 +4,7 @@ layout: doc_page
 
 # Avro
 
-This extension enables Druid to ingest and understand the Apache Avro data format. Make sure to [include](../../operations/including-extensions.html) `druid-avro-extensions` as an extension.
+This extension enables Druid to ingest and understand the Apache Avro data format. Make sure to [include](../../operations/including-extensions.html) `druid-avro-extensions` as an extension.  Inline schema and [Schema Repo](http://schemarepo.org) are supported. [Schema Registry](https://github.com/confluentinc/schema-registry) is not supported.
```

Comment:
i don't think we need to list what we don't support.
also, i would recommend users to mostly use inline schema unless really necessary to do otherwise.